src/clj/rems/service/test_data.clj

```diff
@@ -63,156 +63,172 @@
-                  :sv "Lista med flerval"}
-    :field/optional true
-    :field/options [{:key "Option1"
-                     :label {:en "First option"
-                             :fi "Ensimmäinen vaihtoehto"
-                             :sv "Första alternativ"}}
-                    {:key "Option2"
-                     :label {:en "Second option"
-                             :fi "Toinen vaihtoehto"
-                             :sv "Andra alternativ"}}
-                    {:key "Option3"
-                     :label {:en "Third option"
-                             :fi "Kolmas vaihtoehto"
-                             :sv "Tredje alternativ"}}]}
-
-   {:field/type :table
-    :field/title {:en "Table"
-                  :fi "Taulukko"
-                  :sv "Tabell"}
-    :field/optional true
-    :field/columns [{:key "col1"
-                     :label {:en "First"
-                             :fi "Ensimmäinen"
-                             :sv "Första"}}
-                    {:key "col2"
-                     :label {:en "Second"
-                             :fi "Toinen"
-                             :sv "Andra"}}]}
-
-   {:field/type :label
-    :field/title {:en "The following field types can have a max length."
-                  :fi "Seuraavilla kenttätyypeillä voi olla pituusrajoitus."
-                  :sv "De nästa fälttyperna kan ha bengränsat längd."}
-    :field/optional false}
-
-   ;; fields which support maxlength
-   {:field/type :text
-    :field/title {:en "Text field with max length"
-                  :fi "Tekstikenttä pituusrajalla"
-                  :sv "Textfält med begränsat längd"}
-    :field/optional true
-    :field/max-length 10}
-
-   {:field/type :texta
-    :field/title {:en "Text area with max length"
-                  :fi "Tekstialue pituusrajalla"
-                  :sv "Textområdet med begränsat längd"}
-    :field/optional true
-    :field/max-length 100}
-
-   {:field/type :phone-number
-    :field/title {:en "Phone number"
-                  :fi "Puhelinnumero"
-                  :sv "Telefonnummer"}
-    :field/optional true}
-
-   {:field/type :ip-address
-    :field/title {:en "IP address"
-                  :fi "IP-osoite"
-                  :sv "IP-adress"}
-    :field/optional true}])
+(def all-field-types-example
+  (concat [label-field
+           description-field
+           text-field
+           texta-field
+           header-field
+           (-> date-field (merge {:field/optional true}))
+           (-> email-field (merge {:field/optional true}))
+           (-> attachment-field (merge {:field/optional true}))]
+          conditional-field-example ; array of fields
+          [(-> multiselect-field (merge {:field/optional true}))
+           (-> table-field (merge {:field/optional true}))]
+          max-length-field-example ; array of fields
+          [(-> phone-number-field (merge {:field/optional true}))
+           (-> ip-address-field (merge {:field/optional true}))]))
```

Comment:
We could also use `assoc` which would likely make this a tiny bit simpler.

Reply:
Also a plain `merge` without `->` would work. There's a bunch of other places too.